src/index.html

```diff
@@ -7,12 +7,74 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+
     <link
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
The href attribute in the link tag should point to a CSS file, not an SCSS file. Please change './styles/index.scss' to './styles/index.css'.